packages/shared/src/hooks/source/useSourceActionsFollow.ts

```diff
@@ -20,7 +20,7 @@ export function useSourceActionsFollow({
 }: UseSourceActionsFollowProps): UseSourceActionsFollow {
   const { displayToast } = useToastNotification();
 
-  const { feedSettings } = useFeedSettings();
+  const { feedSettings } = useFeedSettings({ enabled: false });
```

Comment:
```suggestion
  const { feedSettings } = useFeedSettings({ enabled: !!source?.id })
```

this will also make it so feed settings are not loaded on every homepage load